packages/expo-video/ios/VideoManager.swift

```diff
@@ -50,17 +50,36 @@ class VideoManager {
 
   // MARK: - Audio Session Management
 
-  private func switchToActiveAudioSession() throws {
+  internal func setAppropriateAudioSessionOrWarn() {
     let audioSession = AVAudioSession.sharedInstance()
-    try audioSession.setCategory(.playback, mode: .moviePlayback)
-    try audioSession.setActive(true)
-  }
+    var audioSessionCategoryOptions: AVAudioSession.CategoryOptions = []
 
-  internal func switchToActiveAudioSessionOrWarn(warning: String) {
+    let isAnyPlayerPlaying = videoPlayers.allObjects.reduce(false) { result, object in
```

Comment:
maybe you can use `contains(where:)` to avoid traversing the entire list?
https://developer.apple.com/documentation/swift/array/contains(where:)